account/models.py

```diff
@@ -38,10 +38,9 @@ class Account(models.Model):
     def for_request(cls, request):
         user = getattr(request, "user", None)
         if user and user.is_authenticated:
-            try:
-                return Account._default_manager.get(user=user)
-            except Account.DoesNotExist:
-                pass
+            account = user.account
```

Comment:
Won't it raise AttributeError in case of not including app in settings.py for example?

Reply:
If you're not including the app in settings.py, this code should be running anyway (unless you've forgotten to remove the backends and authentication).

Reply:
In case we include app, but not include auth backend, library user will recieve AttributeError, so I guess the better way will be to handle this situation (like checks in django) and inform user what he/she need to do in order for code to work (For e.g.: account not found on user: maybe you forgot to include auth backend).

Reply:
Yeah that makes sense.